### PR TITLE
Add Screen Mockups built with TailwindCSS

### PR DESCRIPTION
**Pull request by yuliyan**
##### Test instructions
* Follow the[ instruction for the initial app setup](https://github.com/Automattic/localized-screenshots/blob/main/README.md) (if not done already).
* Checkout branch locally and start the app with `yarn dev`.
* Go to `/mockups` and test the screen mockups.

##### Screenshots
CleanShot 2023-03-24 at 18 07 41

CleanShot 2023-03-24 at 18 08 20

CleanShot 2023-03-24 at 18 08 51

CleanShot 2023-03-24 at 18 09 14

CleanShot 2023-03-24 at 18 09 35

### CODE REVIEW

File: client/pages/mockups.js [hunk 1]
```diff
@@ -0,0 +1,999 @@
...
+
+function Home() {
+	return (
+		<div className="p-4 text-center">
+			<div className="inline-flex gap-4 items-end p-8 bg-white rounded-md shadow-md text-left">
+				<label className="w-64">
+					<span class="block text-slate-600 text-sm mb-2">Site:</span>
+					<select className="block w-full rounded-md border-slate-300 focus:border-slate-400 focus:ring focus:ring-slate-600 focus:ring-opacity-10">
+						<option>WordPress.com</option>
+						<option>WordPress.com</option>
```
Comment (d-alleyne) [inline client/pages/mockups.js]: Duplicate entry?

File: client/pages/mockups.js [hunk 2]
```diff
@@ -0,0 +1,999 @@
...
+		</div>
+	);
+}
+
+function Home() {
+	return (
+		<div className="p-4 text-center">
+			<div className="inline-flex gap-4 items-end p-8 bg-white rounded-md shadow-md text-left">
+				<label className="w-64">
+					<span class="block text-slate-600 text-sm mb-2">Site:</span>
```
Comment (d-alleyne) [inline client/pages/mockups.js]: Invalid DOM property `class`. Did you mean `className`?

File: client/pages/mockups.js [hunk 3]
```diff
@@ -0,0 +1,999 @@
...
+						<option>WordPress.com</option>
+						<option>WordPress.com</option>
+					</select>
+				</label>
+
+				<label className="w-64">
+					<span class="block text-slate-600 text-sm mb-2">
+						Resolution:
+					</span>
+					<select className="block w-full rounded-md border-slate-300 focus:border-slate-400 focus:ring focus:ring-slate-600 focus:ring-opacity-10">
```
Comment (d-alleyne) [inline client/pages/mockups.js]: Use the `defaultValue` or `value` props on <select> instead of setting `selected` on <option>.

**Comment by d-alleyne** [comment]
On the first screen, I believe we need some form of visual hierarchy. Is the *New Screenshot* button necessary on the first page?

**Comment by dlind1** [comment]
Looks good and it's nice that it's interactive from the start!

The home page and the UX of 'New Screenshot' / 'Browse Screenshots' buttons is a bit inconsistent. What if:
- The home page would display the instructions in an expanded state and the current 'Start session' card with a '-OR- Browse Existing Screenshots' card/button below it, but no buttons in the navigation except for 'Localized Screenshots' homepage link.
- Once in a 'New Screenshot' or 'Browse Screenshots' mode the top navbar could work like it does right now, with the current flow showing as an active button and the other button allowing you to switch.
- The other actions (start from scratch etc.) could then be buttons in the sidebar, once we'd start implementing them.

It might be a good idea to ask a designer for a review as well, I've pinged Saygun for some feedback.

File: client/pages/mockups.js (hunk 2 above)
Comment (yuliyan) [reply client/pages/mockups.js]: Thanks for catching that - it's fixed now.

File: client/pages/mockups.js (hunk 1 above)
Comment (yuliyan) [reply client/pages/mockups.js]: It's just a placeholder.

File: client/pages/mockups.js (hunk 3 above)
Comment (yuliyan) [reply client/pages/mockups.js]: It's supposed to be transformed into a controlled field later when integrated as a component, so I think it's fine to leave it as is for the purpose of the mockup.

**Review by dlind1 — APPROVED** [review]
LGTM!

**Comment by yuliyan** [comment]
Deployed the mockups to fly.io.

**Pull request merged**